internal/resources/cluster/common/advanced_configs.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package common
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	clustercommon "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/cluster/common"
+)
+
+const (
+	advancedConfigurationKey      = "key"
+	advancedConfigurationValueKey = "value"
+)
+
+var AdvancedConfigs = &schema.Schema{
+	Type:        schema.TypeList,
+	Description: "Advanced configuration for TKGm cluster",
```

Comment:
advance configuration of what ?

Reply:
There is a new spec in API for TKGm workload clusters called Advanced Config that has key and value in it. The Description is the same taken from API. It is extra configuration for the workload clusters.

Reply:
if possible can you add more information on what is the advance configuration used for  ?

Reply:
Not got any extra information in the API changes, even checked the PR, let me reach out to the concerned engineer who has done this addition.